### PR TITLE
Added onPress feature

### PR DESCRIPTION
**Pull request by imMayankRaj**
A callback when the user clicks on the toast.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -88,7 +92,7 @@ export default class Toast extends Component {
 
     render() {
         let pos;
-        switch (this.props.position) {
+        switch ('center') {
```
Comment (SiavasFiroozbakht) [inline index.js]: Hi @imMayankRaj, thanks for this pull request. Just looking at the code it seems this change is not required? If we state `'center` in the switch condition the toast will always be shown in the center regardless of user's preference. Should we revert back this bit?

Comment (SiavasFiroozbakht) [reply index.js]: Just verified this on my end and can confirm the tap functionality works on iOS. We still have to revert the line above, but besides that all seems good to me. @crazycodeboy would you be able to look into merging this request?

File: index.js [hunk 2]
```diff
@@ -36,9 +37,12 @@ export default class Toast extends Component {
         }
     }
 
-    show(text, duration, callback) {
+    show(text, duration, callback, onPress) {
         this.duration = typeof duration === 'number' ? duration : DURATION.LENGTH_SHORT;
         this.callback = callback;
+        console.log(typeof onPress)
```
Comment (sesm) [inline index.js]: Is `console.log` necessary here?
Otherwise, good job! 👍

**Comment by ashwinsingh2007** [comment]
This is awesome. I was using this library but recently i wanted to have onPress thing. Can we have this merged asap.

**Pull request merged**